CHANGES

```diff
@@ -1,3 +1,37 @@
+----------------------------------------------------------------
+Released 3.0.0b2 2017-12-11
+
+Changes since 3.0.0b1:
+
+The module `ldap.async` is renamed to `ldap.asyncsearch`, due to
+`async` becoming a keyword in Python 3.7.
+The old module name is deprecated, but will be available as long
+as Python 3.6 is supported.
+
+Lib/
+* Fix error reporting of LDAPObject.set_option()
```

Comment:
`` LDAPObject.set_option()`` was fixed in C code (Modules section)

Reply:
Good catch! Thanks!